### ISSUE TITLE
Support for managing email notifications

### ISSUE THREAD

**Issue by divarvel** [CleverCloud/clever-tools#98]
See #84 

```
clever notify-emails add \
  <name> \
  [--event <event_type>]* \
  [--service <app|addon_id>]* \
  [--send-to <user_id|email_address>]*
```

Thoughts @urcadox @BlackYoup ?

**Comment by urcadox** [comment]
Name is optional so I would use a parameter.

Otherwise, lgtm.

**Comment by divarvel** [comment]
I made it mandatory for webhooks as well, I handle missing names in display but I prefer to make name mandatory in the CLI

**Comment by divarvel** [comment]
Fixed by fd6d52ec551c962f9964ff6a34558e4e82484f80
